pkg/kubelet/kubelet.go

```diff
@@ -2845,6 +2855,117 @@ func (kl *Kubelet) fastStatusUpdateOnce() {
 	}, 100*time.Millisecond, stopCh)
 }
 
+// handleMaxContainerCheckpoints goes through the list of checkpoints of a
+// container in a pod and deletes old checkpoints if there are more than
+// 'kl.maxContainerCheckpoints'.
+func (kl *Kubelet) handleMaxContainerCheckpoints(
+	podUID types.UID,
+	podFullName,
+	containerName string,
+	options *runtimeapi.CheckpointContainerRequest,
```

Comment:
I'd suggest to follow the same formatting style as used in this file(and project). Breaking function signatures this was is not very common. See https://google.github.io/styleguide/go/guide#line-length for more details.